main/src/ca/uwaterloo/flix/api/lsp/provider/CodeActionProvider.scala

```diff
@@ -319,6 +320,35 @@ object CodeActionProvider {
     command = None
   )
 
+  /**
+    * Returns a list of quickfix code action to suggest possibly correct spellings.
+    * Uses Levenshtein Distance to find closed spellings.
+    */
+  private def mkFixMisspelling(qn: Name.QName, loc: SourceLocation, env: Map[String, Symbol.VarSym], uri: String): List[CodeAction] = {
+    if (qn.ident.name.length > 3) {
```

Comment:
I see the constant 3 repeated a lot. Let make it into a local variable. E.g. could be `MinLength`.